### PR TITLE
POM-111 endpoint to provide current user info

### PR DESCRIPTION
**Pull request by sfc-gh-tmalachowski**

### CODE REVIEW

File: src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package pl.gov.coi.pomocua.ads.dev;
+
+import pl.gov.coi.pomocua.ads.UserId;
+import pl.gov.coi.pomocua.ads.users.User;
+import pl.gov.coi.pomocua.ads.users.UsersRepository;
+
+public class FakeUserRepository implements UsersRepository {
```
Comment (sfc-gh-pnescior) [inline src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java:7]: A small thing, but... it's `Fake>User<Repository`, whether the interface is in plural: `UsersRepository`
Maybe let's keep it consistent? (either way)

File: src/test/java/pl/gov/coi/pomocua/ads/users/UserResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    @Autowired TestUsersRepository testUsersRepository;
+
+    @AfterEach
+    void clear() {
+        testUsersRepository.clear();
+    }
+
+    @Test
+    void shouldReturnCurrentUserData() {
```
Comment (sfc-gh-pnescior) [inline src/test/java/pl/gov/coi/pomocua/ads/users/UserResourceTest.java:35]: Maybe another test for missing user and returning HTTP 404?

File: src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    @Override
+    public User getById(UserId userId) {
+        return new User(new FakeCurrentUser().getCurrentUserId(), "fake@email.invalid");
```
Comment (sfc-gh-pnescior) [inline src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java:11]: `FakeUserRepository` is a spring bean, so maybe we could inject `CurrentUser` here?
I know that both of them are plugged in with the same `dev` profile, but `FakeUserRepository` doesn't have to know that

File: src/main/java/pl/gov/coi/pomocua/ads/users/User.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package pl.gov.coi.pomocua.ads.users;
+
+import pl.gov.coi.pomocua.ads.UserId;
+
+public class User {
+    private final UserId userId;
+    public final String email;
```
Comment (sfc-gh-pnescior) [inline src/main/java/pl/gov/coi/pomocua/ads/users/User.java:7]: Maybe let's keep it consistent here? So either all fields public, or private?

File: src/test/java/pl/gov/coi/pomocua/ads/users/UserResourceTest.java (hunk 2 above)
Comment (sfc-gh-tmalachowski) [reply src/test/java/pl/gov/coi/pomocua/ads/users/UserResourceTest.java:35]: good point I'll set the response status to unauthorised as missing user means no account

**Pull request merged**